ulog/doc.go

```diff
@@ -20,10 +20,14 @@
 
 // Package ulog is the Logging package.
```

Comment:
Is it possible to change these autogenerated sentences to make words like "logging" lowercase?